metron-platform/metron-writer/src/main/java/org/apache/metron/writer/BulkWriterComponent.java

```diff
@@ -88,20 +91,21 @@ public void write( String sensorType
     if (!configurations.isEnabled(sensorType)) {
       BulkWriterResponse response = new BulkWriterResponse();
       response.addSuccess(messageId);
-      bulkWriterResponseHandler.handleFlush(sensorType, response);
-      return;
-    }
+      for(FlushPolicy policy: flushPolicies) {
```

Comment:
Notice there is no special handling for the BulkWriterResponseHandler.  Everything is just a policy, so its a bit simpler to test and reason about.